src/main/resources/application.yml

```diff
@@ -1,4 +1,4 @@
-info:
+    info:
```

Comment:
indentation

Reply:
Fixed it - noticed the build weren't happy. Don't even know how that even snuck in 😆